tests/page/page-route.spec.ts

```diff
@@ -167,7 +167,7 @@ it('should properly return navigation response when URL has cookies', async ({ p
 
 it('should override cookie header', async ({ page, server, browserName }) => {
   it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/16773' });
-  it.fail(browserName !== 'firefox');
+  it.fail(browserName !== 'firefox' && browserName !== '_bidiFirefox');
```

Comment:
Note: I'm not sure which pattern should be used in general for those browserName checks which need to accept both firefox and _bidiFirefox, so I kept it simple here, but let me know if we should rather use a regex based check eg `/firefox/i.test(browserName)`

Reply:
Sounds good, done.